test/functional/wallet_assumeutxo.py

```diff
@@ -45,6 +46,19 @@ def setup_network(self):
         self.add_nodes(2)
         self.start_nodes(extra_args=self.extra_args)
 
+    def test_descriptor_import(self, node, wallet_name, key, timestamp, expected_error_message=None):
+        import_request = [{"desc": descsum_create("pkh(" + key.pubkey + ")"),
+                           "timestamp": timestamp,
+                           "label": "Descriptor import test"}]
+        wrpc = node.get_wallet_rpc(wallet_name)
+        result = wrpc.importdescriptors(import_request)
+
+        if expected_error_message is None:
```

Comment:
You can simplify this by just returning the result from the function and then doing these checks respectively in the two places where the function is used now. Should drop the `test_` from the name as well then.